src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
So I guess the better question, is why cant we simply use one pool? Why have two pools in the first place?